src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Sse42.cs

```diff
@@ -17,6 +17,20 @@ public abstract class Sse42 : Sse41
 
         public new static bool IsSupported { get => IsSupported; }
 
+        public abstract class X64 : Sse
```

Comment:
I don't understand why this and `Sse41.X64` inherit from `Sse` and not their containing class, like the rest do.

Reply:
Oops, that is a typo, will fix.